src/map.cpp

```diff
@@ -591,9 +591,9 @@ void Map::transformLiquids(std::map<v3s16, MapBlock*> &modified_blocks,
 		NodeNeighbor sources[6]; // surrounding sources
 		int num_sources = 0;
 		NodeNeighbor flows[6]; // surrounding flowing liquid nodes
-		int num_flows = 0;
+		uint64_t num_flows = 0;
 		NodeNeighbor airs[6]; // surrounding air
-		int num_airs = 0;
+		uint64_t num_airs = 0;
```

Comment:
??

Reply:
https://lgtm.com/projects/g/minetest/minetest/snapshot/edcf968aa097bcd285abc86134a9561f9bd43bac/files/src/map.cpp?sort=name&dir=ASC&mode=heatmap not sure if it fixes the alerts mentioned here, though. It should probably be edited directly, in the loop.

Reply:
https://lgtm.com/rules/1505957036972/